charts/terra-argocd-app/templates/legacy-configs-app.yaml

```diff
@@ -1,35 +1,40 @@
-{{- with .Values -}}
-{{- if .legacyConfigsEnabled -}}
+{{- if eq .Values.releaseType "app" | and .Values.legacyConfigsEnabled -}}
```

Comment:
I think you're trying to do a logical [and](https://helm.sh/docs/chart_template_guide/function_list/#and) here.

Reply:
I am, with pipeline syntax

Reply:
OOOOO, I read this wrong.